src/ui/ResponsiveComponents/ResponsiveTabs.ts

```diff
@@ -211,61 +213,40 @@ export class ResponsiveTabs implements IResponsiveComponent {
     return el.clientWidth < el.scrollWidth;
   }
 
-  private buildDropdownHeader(): Dom {
+  private buildDropdownHeader(): ResponsiveDropdownHeader {
     const dropdownHeader = $$('a', { className: `coveo-dropdown-header ${ResponsiveTabs.TAB_IN_DROPDOWN_HEADER_CSS_CLASS}` });
     const content = $$('p');
     content.text(this.dropdownHeaderLabel);
     const icon = $$('span', { className: 'coveo-more-tabs' }, SVGIcons.icons.arrowDown);
     SVGDom.addClassToSVGInContainer(icon.el, 'coveo-more-tabs-svg');
     content.el.appendChild(icon.el);
     dropdownHeader.el.appendChild(content.el);
-    return dropdownHeader;
-  }
-
-  private bindDropdownHeaderEvents() {
-    const toggle = () => {
-      if (!this.dropdownHeader.hasClass('coveo-dropdown-header-active')) {
-        this.positionPopup();
-        this.dropdownHeader.addClass('coveo-dropdown-header-active');
-      } else {
-        this.closeDropdown();
-      }
-    };
-    new AccessibleButton()
-      .withElement(this.dropdownHeader)
-      .withSelectAction(toggle)
-      .withLabel(this.getDropdownHeaderLabel())
-      .build();
+    return new ResponsiveDropdownHeader('tab', $$(dropdownHeader));
   }
 
   private buildDropdownContent() {
-    const dropdownContent = $$('div', {
+    const dropdownContentElement = $$('div', {
```

Comment:
I would have the "element" inside the variable name if it's not the HTMLElement but the DOM object.